gencred/merge_kubeconfig_secret.py

```diff
@@ -68,19 +92,34 @@ def main(args):
             srcflag = ''
             if args.src_key != args.dest_key:
                 srcflag = '--from-file="%s=%s"' % (args.src_key, orig)
-            call('kubectl create secret generic --namespace "%s" "%s" --from-file="%s=%s" %s --dry-run -oyaml | kubectl replace -f -' % (args.namespace, args.name, args.dest_key, merged, srcflag)) #pylint: disable=line-too-long
+            call('kubectl --context="%s" create secret generic --namespace "%s" "%s" --from-file="%s=%s" %s --dry-run -oyaml | kubectl replace -f -' % (args.context, args.namespace, args.name, args.dest_key, merged, srcflag)) #pylint: disable=line-too-long
         else:
             content = ''
             with open(merged, 'r') as mergedFile:
                 yamlPad = '    '
                 content = yamlPad + mergedFile.read()
                 content = content.replace('\n', '\n' + yamlPad)
-            call('kubectl patch --namespace "%s" "secret/%s" --patch "stringData:\n  %s: |\n%s\n"' % (args.namespace, args.name, args.dest_key, content)) #pylint: disable=line-too-long
+            call('kubectl --context="%s" patch --namespace "%s" "secret/%s" --patch "stringData:\n  %s: |\n%s\n"' % (args.context, args.namespace, args.name, args.dest_key, content)) #pylint: disable=line-too-long
```

Comment:
I think large templates like this should make it clear what arguments it's taking with a line break.
```
call('very long template'
    % (arg1, arg2))
```